tensorboard/data/server/reservoir.rs

```diff
@@ -307,52 +357,80 @@ mod tests {
         rsv.offer(Step(0), "zero");
         rsv.offer(Step(1), "one");
         rsv.offer(Step(2), "two");
-        rsv.offer(Step(3), "three");
+        rsv.offer(Step(3), "three???"); // this funny-looking record will be preempted
```

Comment:
I hadn’t considered that, only because I originally wrote the reservoir
with preemption from the beginning, and wrote the test all at once. But
on revisiting it I agree with your assessment. I also think that the
extensions to the property-based test are pretty reasonable to cover
normal cases of preemption, and we now have extra tests for more edge
cases. So I’ve gone and just stripped the preemption from this test.
Work for you?